### PR TITLE
Issue #8 (Add link to certification and training programs.)

### PR DESCRIPTION
**Pull request by lloyd-pinrun**
There is now a link at the bottom of the career assessment results page.  Check templates/careers.mustache

### CODE REVIEW

File: templates/careers.mustache [hunk 1]
```diff
@@ -39,6 +39,7 @@
           <h2>Get Started Now</h2>
           <p><strong>An Opportunity Center can help you get started on one of these career paths.</strong></p>
           <p>Sign up for the Opportunity Center job system and receive a personalized recommendation for next steps and a follow-up from a career advisor.</p>
+          <p><strong>Find information on certification and training programs <span class="link"><a href="https://careerpathnola.business/training-and-educucation" target="_blank">here</a></span>.</strong></p>
```
Comment (nihonjinrxs) [inline templates/careers.mustache]: You'll want to link something more descriptive than `here` for accessibility reasons.

You can reference the [Web Content Accessibility Guidelines (WCAG) 2.0](https://www.w3.org/WAI/WCAG20/quickref/) for widely accepted standards on interface accessibility.

For this particular issue, you'll want to read about [WCAG 2.0 Technique H30: Providing link text that describes the purpose of a link for anchor elements](https://www.w3.org/TR/WCAG20-TECHS/H30.html).

**Comment by nihonjinrxs** [comment]
@cam-carter Is there a reason this PR is on the codeforamerica repo instead of our local fork?

**Comment by lloyd-pinrun** [comment]
@nihonjinrxs Just noticed this! My bad!

**Comment by nihonjinrxs** [comment]
@cam-carter Since you've moved this over to our fork, you can close this PR.

**Pull request closed without merging**